rust/src/storage/inmemory/notus_advisory.rs

```diff
@@ -0,0 +1,16 @@
+// SPDX-FileCopyrightText: 2025 Greenbone AG
+//
+// SPDX-License-Identifier: GPL-2.0-or-later WITH x11vnc-openssl-exception
+
+use crate::storage::{
+    dispatch::Dispatcher, error::StorageError, items::notus_advisory::NotusAdvisory,
+};
+
+use super::InMemoryStorage;
+
+impl Dispatcher<()> for InMemoryStorage {
```

Comment:
I tried to like the approach of having each implementation in its own file, but it really breaks my flow when I try to look something up. I constantly have to jump around, and when I do, I need to remember that the actual implementation is in the main impl, not the trait impl, which means I have to jump back again.